examples/pipe/src/pipe.clj

```diff
@@ -36,60 +31,28 @@
   topology builder."
   [builder]
   (-> (j/kstream builder (topic-config "input"))
-      (j/peek (fn [[k v]]
-                (info (str {:key k :value v}))))
+      (j/peek (fn [[k v]] (println (str {:key k :value v}))))
       (j/to (topic-config "output")))
   builder)
 
 (defn start-app
   "Starts the stream processing application."
-  [app-config]
-  (let [builder (j/streams-builder)
-        topology (build-topology builder)
-        app (j/kafka-streams topology app-config)]
-    (j/start app)
-    (info "pipe is up")
-    app))
+  ([] (start-app app-config))
+  ([app-config]
+   (let [builder (j/streams-builder)
+         topology (build-topology builder)
+         app (j/kafka-streams topology app-config)]
+     (j/start app)
+     (println "pipe is up")
```

Comment:
I kept getting warnings that the logging is not setup, therefore, given this is a sample project, I replaced it with `println`s.